services/server/test/integration/serviceInstance.js

```diff
@@ -31,6 +31,22 @@ test('GET /instance returns current serviceInstance', async (t) => {
   t.is(newServiceInstance.pingCount, serviceInstance.pingCount + 1);
 });
 
+test('GET /instance/date returns current time within buffer', async (t) => {
+  const { api } = t.context.app;
+
+  const requestTimestampMS = Date.now();
+  const { data: { date }, status } = await api.get('/instance/date');
+  t.is(status, StatusCodes.OK);
+
+  const bufferMS = 500;
+
+  const lowerBound = requestTimestampMS;
+  const upperBound = requestTimestampMS + bufferMS;
+
+  t.truthy(date >= lowerBound);
```

Comment:
`t.true` seems like a better fit here. Also would be nice for the assertion string to include the values of `date, lowerBound, and upperBound` (e.g. `t.true(bool, "date=, lowerBound=, upperBound="`)

Reply:
done